src/client/address_book/storage.h

```diff
@@ -54,6 +54,22 @@ namespace client {
 /// @class AddressBookDefaults
 /// @brief Default string constants used throughout address book
 struct AddressBookDefaults {
+  /// @enum Subscription
+  /// @brief Subscription type for where to load/save subscription addresses
+  /// @notes Scoped to prevent namespace pollution (otherwise, purely stylistic)
+  enum struct Subscription
+  {
+    Default,
+    User,
+    Private,
+  };
+
+  /// @alias AddressMap
+  /// @brief Maps human-readable hostname to an identity hash and subscription type
+  /// @notes Alias makes refactoring and use simpler
+  using AddressMap =
```

Comment:
https://google.github.io/styleguide/cppguide.html#Aliases

You'll need to decide how you intend to use this code in the future.

Reply:
Ok, I'll add a `details` doxygen comment with intended use, and get rid of the vague `notes` comment.